backend/src/create.rs

```diff
@@ -166,6 +153,13 @@ pub fn create_from_osm(
         })
         .collect();
 
+    // Copy all the fields
+    let intersections: Vec<Intersection> = graph
+        .intersections
+        .into_values()
+        .map(|i| Intersection::from_graph(i, &roads))
```

Comment:
Intersection roads are now sorted clockwise from north, so this got changed around a little.

This was necessary to sanely "split" the roads coming into an intersection into groups of adjacent roads.